### PR TITLE
[13.0][FIX] dms: Fix multi-company rules + migration scripts

### PR DESCRIPTION
**Pull request by victoralmau**
Fix multi-company rules + migration scripts.

Please @Yajo and @pedrobaeza can you review it?

It's necessary to aplly to 14.0 too.

@Tecnativa TT30826

### CODE REVIEW

**Comment by pedrobaeza** [comment]
A migration script is needed. See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

File: dms/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Document Management System",
     "summary": """Document Management System for Odoo""",
-    "version": "13.0.6.0.0",
+    "version": "13.0.7.0.0",
```
Comment (pedrobaeza) [inline dms/__manifest__.py]: Don't change the major version, but minor one. The change is not of such entity.

File: dms/static/description/index.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-<meta name="generator" content="Docutils 0.15.1: http://docutils.sourceforge.net/" />
+<meta name="generator" content="Docutils: http://docutils.sourceforge.net/" />
```
Comment (pedrobaeza) [inline dms/static/description/index.html]: Don't change this

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Fast-tracking it as straight-forward (and needed):

/ocabot merge nobump

**Review by yajo — APPROVED** [review]

**Pull request merged**